plugins/CoreHome/stylesheets/coreHome.less

```diff
@@ -265,3 +265,98 @@ a.Piwik_Popover_Error_Back {
         margin-top: 5px;
     }
 }
+
+.copyToClipboardIcon {
+    font-family: 'matomo' !important;
+    font-style: normal;
+    font-variant: normal;
+    text-transform: none;
+    font-size: 1.6em;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    display:inline-block;
+    margin-left:4px;
+    float:left;
+}
+
+.copyToClipboardIcon:before {
+    content: "\e905";
+}
+
+.copyToClipboardIconCheck {
+    font-family: 'matomo' !important;
+    font-style: normal;
+    font-weight: 400;
+    font-variant: normal;
+    text-transform: none;
+    font-size: 1.6em;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    display:inline-block;
+    margin-left:4px;
+    float:left;
+}
+
+.copyToClipboardIconCheck:before {
+    content: "\e906";
+    color: #43a047 !important;
+}
+
+.copyToClipboardButton {
+    position: absolute;
+    right: .5em;
+    top: .5em;
+    margin-right: 0.2em;
+    margin-top: 0.2em;
+    border-radius: 100%;
+    width: 3.0em;
+    height: 3.0em;
+    display: inline-block;
+    transition: width 0.7s;
+    overflow: hidden;
+    background: #fefefe;
+    padding-left:4px;
+    padding-right:4px;
+    cursor: pointer;
+}
+
+.copyToClipboardCopiedDiv {
+    font-size:0.8em;
+    display: none;
+    position: absolute;
+    right: 9em;
+    top: 1.6em;
+    margin-right: 1em;
+    padding-top:0.4em;
+    width: 10.2em;
+    height: 2em;
+    border-radius: 5%;
+    background: #fefefe;
+    text-align: center;
+    vertical-align: middle;
+}
+
+.copyToClipboardButton:hover span {
```

Comment:
We should also set the background color for `:focus`. The current color while clicking the button looks ugly. Guess that colors is set by materialize, so we maybe should unset the color for all buttons/selects.

Reply:
I've overridden the button focus color, which definitely looks better than the pale green(?) materialize default. I'm not sure I want to change this application wide without some sort of design input, so that might be something for a separate ticket.